test/wormhole.test.ts

```diff
@@ -452,6 +463,123 @@ export function runWormholeTests(domain: string, setupDomain: DomainSetupFunctio
       })
     })
 
+    describe('relay', () => {
+      let relay: BasicRelay
+      const expiry = '10000000000'
+      const gasFee = parseEther('1.0')
+
+      before(async () => {
```

Comment:
I'd use `beforeEach` blocks and drop finally clauses in test cases

Reply:
In general I agree but in this case the Rinkeby test setup is painfully slow and expensive (and the main Rinkeby faucet has been offline for weeks). So until Arbitrum supports running a local blockchain (or until the Rinkeby faucet is back online) I'd suggest to keep this structure.

Reply:
ahh okay... Can you perhpaps drop a comment why we do things this way then?

Reply:
yes, sure